src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
 
     export function requestAnimationFramePolyfill(fn: () => void) {
```

Comment:
It needs to be exported since the `Timeout` RenderPolicy uses it as the default timeout